README.md

```diff
@@ -26,98 +26,125 @@ For Production, for the time being, please checkout the repository and build/pus
 
 ## Requirements
 
-Make sure you have `docker` & `docker compose` installed on the server / system. Both are installed by most docker utilities, including Docker Desktop and Rancher Desktop.
+Make sure you have `docker` & `docker-compose` installed on the server / system. Both are installed by most docker utilities, including Docker Desktop and Rancher Desktop.
 
-Note: `docker compose` without the hyphen is now the primary method of using docker-compose, per the Docker documentation.
+Note: `docker-compose` without the hyphen is now the primary method of using docker-compose, per the Docker documentation.

```

Comment:
```suggestion
Note: `docker compose` without the hyphen is now the primary method of using docker-compose, per the Docker documentation.

```
Everyone may have a different opinion, but as I see it, `docker compose` should be better since we are putting this with hyphen everywhere else.

Reply:
You're probably right but I've seen with hyphen being more consistently supported. I've also not seen success with using `docker compose` over `docker-compose` I'm not sure if it's consistently enabled by default. I'm possibly wrong about this.